webui/react/src/e2e/tests/experimentList.spec.ts

```
@@ -559,4 +561,84 @@ test.describe('Experiment List', () => {
       });
     });
   });
+
+  test.describe('Row Actions', () => {
+    let destinationProject: V1Project;
+    let destinationWorkspace: V1Workspace;
+    let experimentId: number;
+
+    // create a new project, workspace and experiment
+    test.beforeAll(
+      async ({
+        backgroundApiProject,
+        backgroundApiWorkspace,
+        newProject: {
+          response: { project },
+        },
+      }) => {
+        destinationWorkspace = (
+          await backgroundApiWorkspace.createWorkspace(backgroundApiWorkspace.new())
+        ).workspace;
+        destinationProject = (
+          await backgroundApiProject.createProject(
+            project.workspaceId,
+            backgroundApiProject.new({ projectProps: { workspaceId: project.workspaceId } }),
+          )
+        ).project;
+
+        const expId = Number(
+          detExecSync(
+            `experiment create ${fullPath('examples/tutorials/mnist_pytorch/adaptive.yaml')} --paused --project_id ${project.id}`,
+          ).split(' ')[2],
+        ); // returns in the format "Created experiment <exp_id>"
+
+        if (!Number.isNaN(expId)) experimentId = expId;
```

throw error instead of just skipping assignment?